potodo/interactive.py

```diff
@@ -65,6 +74,9 @@ def _confirmation_menu(choosen_file: str, directory: str) -> int:
 def get_dir_list(
     repo_path: Path, exclude: Iterable[Path], ignore_matches: Callable[[str], bool]
 ) -> List[str]:
+    """
+    Return a list of directories in a directory.
```

Comment:
This returns the list of directories containing `.po` files insid the `repo_path` directory, ignoring the folders in `exclude` and the folder matching the function `ignore_matching`